doc/classes/GraphEdit.xml

```diff
@@ -7,6 +7,7 @@
 		[GraphEdit] provides tools for creation, manipulation, and display of various graphs. Its main purpose in the engine is to power the visual programming systems, such as visual shaders, but it is also available for use in user projects.
 		[GraphEdit] by itself is only an empty container, representing an infinite grid where [GraphNode]s can be placed. Each [GraphNode] represents a node in the graph, a single unit of data in the connected scheme. [GraphEdit], in turn, helps to control various interactions with nodes and between nodes. When the user attempts to connect, disconnect, or delete a [GraphNode], a signal is emitted in the [GraphEdit], but no action is taken by default. It is the responsibility of the programmer utilizing this control to implement the necessary logic to determine how each request should be handled.
 		[b]Performance:[/b] It is greatly advised to enable low-processor usage mode (see [member OS.low_processor_usage_mode]) when using GraphEdits.
+		[b]Note:[/b] Keep in mind that [method Node.get_children] will also return the connection layer node due to technical limitations.
```

Comment:
```suggestion
		[b]Note:[/b] Keep in mind that [method Node.get_children] will also return the connection layer node named [code]_collision_layer[/code] due to technical limitations. This behavior may change in future releases.
```

Reply:
Well, connections with collisions would certainly be interesting 😆